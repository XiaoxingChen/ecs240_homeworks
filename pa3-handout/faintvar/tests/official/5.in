c Branches
p 3 5 6 1 5
c bb1: ;
b 1
e 1 2
e 1 3
e 1 4
c bb2: v1 = v2;
b 2 1 2
e 2 5
c bb3: v3 = v2;
b 3 3 2
e 3 5
c bb4: ;
b 4
e 4 5
c bb5: print(v3);
b 5 0 3
