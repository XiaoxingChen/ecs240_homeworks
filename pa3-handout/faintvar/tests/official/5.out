fvin 1 1
fvin 2 1 2
fvin 3 1 3
fvin 4 1 2
fvin 5 1 2
